packages/cli/src/commands/build/index.ts

```diff
@@ -826,6 +832,43 @@ function mergeFlags(
   });
 }
 
+/**
+ * Takes the build output and writes all the variants into the `variants.json`
+ * file. It'll skip variants that already exist.
+ */
+async function mergeVariants(
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** rename to something like `writeVariantsJson`

This would make it clearer that (1) this doesn't work like the other `merge*` functions here and (2) is writing out a separate file, not building up data for the main `config.json` file.‏

Reply:
Sounds good, done!